class.jetpack.php

```diff
@@ -4498,7 +4498,7 @@ public function validate_remote_register_response( $response ) {
 			return new Jetpack_Error( 'jetpack_id', sprintf( __( 'Error Details: Jetpack ID begins with a numeral. Do not publicly post this error message! %s', 'jetpack' ) , $entity ), $entity );
 		}
 
-	    return true;
+	    return $json;
```

Comment:
Since we're changing the return for this method, can we update the docblock at the top of the function?